### ISSUE TITLE
Accessibility and admin setting tabs have inverted visibility

### ISSUE THREAD

**Issue by UnicornOnLSD** [RMC-14/RMC-14#6730]
## Description

Regular players can see the admin setting tab, admins can see both admin and accessibility

**Reproduction**

Be admin
Able to see accessibility
Be player
Unable to see accessibility tab, uh oh !
**Screenshots**

What Noct, admin sees:

![Image](https://github.com/user-attachments/assets/5a05543f-2f66-4d0d-941d-e8ea8c84a6fa)

What I, regular player sees:

![Image](https://github.com/user-attachments/assets/474fd2b4-df46-400f-8e14-b8c149c2e3ec)

**Additional context**

Gordon the cake is a lie

**Comment by PursuitInAshes** [comment]
Unable to replicate, probably got fixed sometime recently and nobody closed the issue
